cocos2d/physics/CCPhysicsSpriteWebGLRenderCmd.js

```diff
@@ -80,8 +80,20 @@
         return this._transform;
     };
 
+    proto._updateDirty = function(child){
+        var cmd = child._renderCmd;
+        if(cmd)
+            cmd._dirty = true;
```

Comment:
Not all render command have `_dirty` property. Please use some common public API, do not access the private properties directly